unpackaged/post/config/objects/hed__Education_History__c.object

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <actionOverrides>
+        <actionName>View</actionName>
+        <comment>Action override created by Lightning App Builder during activation.</comment>
+        <content>K12_Education_History_Record_Page</content>
+        <formFactor>Large</formFactor>
+        <skipRecordTypeSelect>false</skipRecordTypeSelect>
+        <type>Flexipage</type>
+    </actionOverrides>
+    <compactLayoutAssignment>K12_Kit_Education_History_Compact_Layout</compactLayoutAssignment>
+    <compactLayouts>
+        <fullName>K12_Kit_Education_History_Compact_Layout</fullName>
+        <fields>Name</fields>
+        <fields>hed__Account__c</fields>
+        <fields>hed__Contact__c</fields>
+        <fields>School_Enrollment_Status__c</fields>
```

Comment:
@m-kinkade This is a field managed by K-12 Architecture kit. When referencing it in unmanaged metadata, please be certain to prepend the namespace token so it will work in both unmanaged and managed contexts for the source directory.